backend/drive/drive.go

```diff
@@ -1086,15 +1086,12 @@ func getClient(ctx context.Context, opt *Options) *http.Client {
 
 func getServiceAccountClient(ctx context.Context, opt *Options, credentialsData []byte) (*http.Client, error) {
 	scopes := driveScopes(opt.Scope)
-	conf, err := google.JWTConfigFromJSON(credentialsData, scopes...)
+	credentials, err := google.CredentialsFromJSON(ctx, credentialsData, scopes...)
 	if err != nil {
 		return nil, fmt.Errorf("error processing credentials: %w", err)
 	}
-	if opt.Impersonate != "" {
```

Comment:
Removing this is a problem though as lots of rclone users use high priviledge service accounts with `--drive-impersonate user@example.com`.

Maybe there is a better way of passing this in?